src/memory/binding.rs

```diff
@@ -1311,11 +1310,20 @@ pub(crate) enum MemoryBindingOperation {
 /// Memory binding policy
 ///
 /// Not all systems support all kinds of binding.
-/// [`Topology::feature_support()`] may be used to query the
-/// actual memory binding support in the currently used operating system.
-#[cfg_attr(any(test, feature = "proptest"), derive(Sequence))]
+/// [`Topology::feature_support()`] may be used to query the actual memory
+/// binding support in the currently used operating system.
 #[derive(
-    Copy, Clone, Debug, Default, Display, Eq, Hash, IntoPrimitive, PartialEq, TryFromPrimitive,
+    Copy,
+    Clone,
+    Debug,
+    Default,
+    Display,
+    Eq,
+    Hash,
+    IntoPrimitive,
+    PartialEq,
+    TryFromPrimitive,
+    Sequence,
```

Comment:
enum_iterator is now an unconditional dependency, so derive(Sequence) is unconditional.